src/main/java/tr/edu/ogu/ceng/dao/InternshipRegistryRepository.java

```diff
@@ -1,13 +1,13 @@
 package tr.edu.ogu.ceng.dao;
 
 import org.springframework.data.domain.Page;
-import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 import tr.edu.ogu.ceng.model.InternshipRegistry;
 
 public interface InternshipRegistryRepository extends JpaRepository<InternshipRegistry, Long> {
 
-	Page<InternshipRegistry> findAllById(Long Id, PageRequest pageable);
+	Page<InternshipRegistry> findAllById(Long internshipId, Pageable pageable);
```

Comment:
Fonksiyonun doğru mantıkta çalışabilmesi için method ismi findAllByInternshipId olmalı.

Reply:
gerekli düzenleme yapıldı. @yunusakyol5